machine_management/applying-autoscaling.adoc

```diff
@@ -5,29 +5,25 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Applying autoscaling to an {product-title} cluster involves deploying a
-ClusterAutoscaler and then deploying MachineAutoscalers for each Machine type
-in your cluster.
+Applying autoscaling to an {product-title} cluster involves deploying a cluster autoscaler and then deploying machine autoscalers for each Machine type in your cluster.
```

Comment:
Should Machine here be machine?

Reply:
Yeah probably. Since this PR has already been merged, I've opened a new PR [1] to make this change. Thanks @abrennan89!

@jeana-redhat do you mind doing the peer review on that one?

[1] https://github.com/openshift/openshift-docs/pull/28431

Reply:
@jeana-redhat Also, are there still CPs pending on this PR for 4.4 and 4.5?

Reply:
Just closing the loop in this thread:
Ashleigh, good eye!
Andrea, thanks for the fix and your PR looks great. Cherry picking finally done as of this AM.
Phew!